data/phi0631/phi001/phi0631.phi001.perseus-lat3.xml

```diff
@@ -28,15 +28,16 @@
               <sourceDesc>
                 <biblStruct>
                     <monogr>
-                        <author>C. Sallusti Crispi</author>
-                    	<title>Catilina, Iugurtha, Orationes Et Epistulae Excerptae De Historiis</title>
-                    	<editor role="editor" n="Ahlberg">Axel W. Ahlberg</editor>
+                        <author>Sallust</author>
```

Comment:
The full name is fine here since this should match the print. We don't enforce name changes in the monograph section, we only regularize at the top of the header.